test/functional/query-builder/select/query-builder-select.ts

```diff
@@ -27,6 +27,21 @@ describe("query builder > select", () => {
             "FROM post post");
     })));
 
+    it("should append all entity mapped columns from main selection to SELECT DISTINCT statement", () => Promise.all(connections.map(async connection => {
+        const sql = connection.manager.createQueryBuilder(Post, "post")
+            .distinct()
+            .disableEscaping()
+            .getSql();
+
+        expect(sql).to.equal("SELECT DISTINCT post.id AS post_id, " +
```

Comment:
In general we should try to not compare query text in our tests, but the behavior.  If for some reason we would change default naming strategy in the future we would have to modify many tests. However this(wrong) approach is used in this file, so I'm ok with it here.